### PR TITLE
Fix/google oauth

### PR DESCRIPTION
**Pull request by ZarinaPoyet**

### CODE REVIEW

File: client/src/hooks/useAuth.ts [hunk 1]
```diff
@@ -50,6 +66,10 @@ const useAuth = () => {
   const handleUserInfo = async () => {
     try {
       const name = cookies['app-username'];
+      if (!name) {
```
Comment (jianingroja) [inline client/src/hooks/useAuth.ts:69]: we don't need this conditional check

File: server/controllers/users.ts [hunk 2]
```diff
@@ -110,6 +110,18 @@ export const googleLogin = async (req: Request, res: Response) => {
...
+      httpOnly: true,
+    });
+
+    res.cookie('username', user.username, {
+      domain: 'localhost',
+      path: '/',
+      httpOnly: true,
+    });
+
     return res.status(200).json({ user: user, token: newSessionToken });
```
Comment (jianingroja) [inline server/controllers/users.ts:125]: do we need to send back the token? since it is stored in cookie

**Review by jianingroja — COMMENTED** [review]
clean up the console logs and commented code:)

File: server/controllers/users.ts (hunk 2 above)
Comment (ZarinaPoyet) [reply server/controllers/users.ts:125]: yes

**Pull request merged**